nginx-proxy/patch-local-cors.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
Please use `env` so we can use these scripts on NixOS:
```suggestion
#!/usr/bin/env sh
```